### PR TITLE
Syndicate Lavaland - Atmospheric Research Edition

### PR DESCRIPTION
**Pull request by Addust**
# Document the changes in your pull request

Syndicate Lavaland is now for atmospheric research, with its own custom HFR boxes

# Why is this good for the game?
bup asked me to do it
also it means you can now learn PROPER, REAL ATMOS SCIENCE!!!!

# Testing

# Spriting
sprites from bup and Willymount00

# Wiki Documentation

change picture and shit

# Changelog

:cl:  
rscadd: Syndicate Lavaland has been refitted for gas-related research instead of biological warfare.
rscadd: Thanks to Bup and Willymount00 for the sprites used in the syndie gamer HFR
/:cl:

### CODE REVIEW

**Review by cuackles — APPROVED** [review]
In line with the current donut so it's good

**Comment by Addust** [comment]
> In line with the current donut so it's good

my brother in ratvar i made the donut

anyways when merge

**Review by neeshacark — APPROVED** [review]
how long has this been in pr limbo holy shit

maps are fine... i think? im not pipe boy, but it looks fine to me

File: code/game/objects/items/circuitboards/machine_circuitboards.dm [hunk 1]
```diff
@@ -401,20 +401,29 @@
...
+	build_path = /obj/machinery/atmospherics/components/unary/hypertorus/fuel_input/syndicate
+
 /obj/item/circuitboard/machine/HFR_waste_output
 	name = "HFR Waste Output (Machine Board)"
 	greyscale_colors = CIRCUIT_COLOR_ENGINEERING
 	build_path = /obj/machinery/atmospherics/components/unary/hypertorus/waste_output
 	req_components = list(
 		/obj/item/stack/sheet/plasteel = 5)
 
+/obj/item/circuitboard/machine/HFR_waste_input/syndicate
```
Comment (ynot01) [inline code/game/objects/items/circuitboards/machine_circuitboards.dm]: `HFR_waste_input` does not seem correct here

File: code/modules/atmospherics/machinery/components/fusion/hfr_core.dm [hunk 2]
```diff
@@ -169,6 +169,15 @@
 	///Flags used in the alert proc to select what messages to show when the HFR is delaminating (HYPERTORUS_FLAG_HIGH_POWER_DAMAGE | HYPERTORUS_FLAG_HIGH_FUEL_MIX_MOLE | HYPERTORUS_FLAG_IRON_CONTENT_DAMAGE | HYPERTORUS_FLAG_IRON_CONTENT_INCREASE | HYPERTORUS_FLAG_EMPED)
 	var/warning_damage_flags = NONE
 
+/obj/machinery/atmospherics/components/unary/hypertorus/core/syndicate
+	desc = "This is the Hypertorus Fusion Reactor core, an advanced piece of technology to finely tune the reaction inside of the machine. It has I/O for cooling gases. This one has been modified by the Syndicate, both to look cool and only transmit on Syndicate frequencies."
+	radio_key = /obj/item/encryptionkey/syndicate
+	syndicate_machine = 1
```
Comment (ynot01) [inline code/modules/atmospherics/machinery/components/fusion/hfr_core.dm]: should this not be TRUE?

File: code/game/objects/items/circuitboards/machine_circuitboards.dm (hunk 1 above)
Comment (Addust) [reply code/game/objects/items/circuitboards/machine_circuitboards.dm]: i dont have vscode on the computer im currently on but i should be able to webedit it

**Comment by Addust** [comment]
get this outta limbo i webedited the trouble away

**Review by AMyriad — CHANGES_REQUESTED** [review]
Closing this sometime soon if map conflicts aren't fixed

**Comment by Addust** [comment]
> Closing this sometime soon if map conflicts aren't fixed

i'm on holiday and the conflicts are on the scale of "adding fucking access helpers", closing it myself so that i can reopen it once its fixed

**Pull request closed without merging**